pages/CaseInformation.vue

```diff
@@ -0,0 +1,185 @@
+<template>
+  <loading-message v-if="loading" />
+  <map-component
+    v-else
+    :pageInfo="pageInfo"
+    :originalMarkers="originalMarkers"
+    :filtersGeneralInfo="filtersGeneralInfo"
+    :filterProperties="filterProperties"
+    :includeTimeline="false"
+    iconBehaviour="constant"
+  >
+  </map-component>
+</template>
+
+<script>
+import { SPARQLQueryDispatcher } from "~/assets/js/SPARQLQueryDispatcher";
+import APIDataHandler from "~/assets/js/APIDataHandler";
+import json from "../big-query-output.json";
+import MapComponent from "../components/MapComponent.vue";
+import LoadingMessage from "../components/LoadingMessage.vue";
+import Swal from "sweetalert2";
+
+definePageMeta({
+   layout: 'default'
+ })
+
+export default {
+  components: { MapComponent, LoadingMessage },
+  data: () => ({
+    pageInfo: {
+      title: "Witches Meetings",
+      html: '<div>This map shows the geographical residence location for each accused witch in Scotland taken from the Survey of Scottish Witchcraft Database. Out of the <b class="font-bold">3212</b> accused witches whose names are known, the residence for <b class="font-bold">3142</b> witches has been located. The majority of the residences are accurately located down to the precise settlement, while others range from parish to county depending on the records surviving for each accused witch. There is a total of 821 different locations recorded in the database; all but 25 of these have been identified. The remaining unidentified place-names are currently recorded as \‘County of’\ on the map.</div>',
+      footer: "witches.is.ed.ac.uk",
+      confirmButtonText: "Close",
+      type: "info",
+      showCloseButton: true,
+    },
+    queryOutput: json,
+    sparqlUrl: "https://query.wikidata.org/sparql",
+    wikiPages: [],
+    loading: true,
+    originalMarkers: [],
+    filtersToFind: [
+      ["demonicPact", "constant"],
```

Comment:
might be the case that prettier has been run over this already but if not, could we give it a single pass? Thanks Ruby

Reply:
done

Reply:
Thank you